mkt/settings.py

```diff
@@ -298,7 +298,8 @@ def path(*args):
 APP_DELETION_EMAIL = 'marketplace-staff+deletions@mozilla.org'
 SUPPORT_GROUP = 'https://groups.google.com/forum/#!forum/mozilla.appreview'
 MARKETPLACE_EMAIL = 'marketplace-staff@mozilla.org'
-MKT_FEEDBACK_EMAIL = 'apps-feedback@mozilla.com'
+MKT_APPS_FEEDBACK_EMAIL = 'apps-feedback@mozilla.com'
```

Comment:
It's certainly messy , but it needs to stay since it's the email used for general Marketplace feedback (the email is misleading :/).